WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -722,14 +724,18 @@ class MySiteViewModel @Inject constructor(
     @Suppress("EmptyFunctionBlock")
     private fun onGetMoreViewsClick() {
         cardsTracker.trackTodaysStatsCardGetMoreViewsNudgeClicked()
-        _onNavigation.value = Event(
-            SiteNavigationAction.OpenTodaysStatsGetMoreViewsExternalUrl(URL_GET_MORE_VIEWS_AND_TRAFFIC)
-        )
+        if (jetpackFeatureRemovalPhaseHelper.shouldShowStaticPage()) {
+            _onNavigation.value = Event(SiteNavigationAction.ShowJetpackRemovalStaticPostersView)
+        } else {
+            _onNavigation.value = Event(
+                SiteNavigationAction.OpenTodaysStatsGetMoreViewsExternalUrl(URL_GET_MORE_VIEWS_AND_TRAFFIC)
+            )
+        }
     }
 
     private fun onTodaysStatsCardFooterLinkClick() {
         cardsTracker.trackTodaysStatsCardFooterLinkClicked()
-        navigateToTodaysStats()
+         navigateToTodaysStats()
```

Comment:
np: This feels off indented and not a needed change

Reply:
Fixed in commit [2cd935d](https://github.com/wordpress-mobile/WordPress-Android/pull/18120/commits/2cd935daacd78fe2c7f2ddc9720b415d68b797ff)